src/mplfinance/plotting.py

```diff
@@ -1055,6 +1063,9 @@ def _valid_addplot_kwargs():
 
         'yscale'      : { 'Default'     : None,
                           'Validator'   : lambda value: _yscale_validator(value) },
+
+        'where'      : { 'Default'     : 'pre',
```

Comment:
I am somewhat inclinded to call this `stepwhere` to avoid confusion with other matplotlib apis that may be accessible through `make_addplot()` kwargs (for example `fill_between`, currently in development for addplot, which also has a `where` kwarg that has a very different meaning).  If so, then possibly also change `valid_wheres` to `valid_stepwheres`.